shared/chat/conversation/search/index.desktop.js

```diff
@@ -3,69 +3,181 @@
+        <Kb.Avatar username={item.author} size={24} />
+        <Kb.Text type="Body" style={styles.hitSummary}>
+          {item.summary}
+        </Kb.Text>
+        <Kb.Text type="BodySmall" style={styles.time}>
+          {formatTimeForMessages(item.timestamp)}
+        </Kb.Text>
+      </Kb.ClickableBox>
+    )
+  }
+
+  _onChangedText = (text: string) => {
+    this._text = text
+  }
+
+  _onFocus = () => {
+    this.setState({showResultList: true})
+  }
+
+  _onBlur = () => {
+    setTimeout(() => this.setState({showResultList: false}), 300)
+  }
+
+  _onKeydown = e => {
+    if (e.key === 'Escape') {
+      this.props.selfHide()
+    }
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.hits.length === 0 && this.props.hits.length > 0) {
+      this.props.loadSearchHit(0)
+      this.setState({selectedIndex: 0})
+    }
   }
+
   render() {
     return (
-      <Kb.Box2 direction="horizontal" style={styles.outerContainer} fullWidth={true} gap="tiny">
-        <Kb.Box2 direction="horizontal" style={styles.inputContainer} fullWidth={true}>
-          <Kb.Box2 direction="horizontal" gap="xtiny" fullWidth={true} centerChildren={true}>
-            <Kb.Icon type="iconfont-search" color={Styles.globalColors.black_50} fontSize={16} />
-            <Kb.Input
-              hideUnderline={true}
-              hintText="Search..."
-              small={true}
-              uncontrolled={true}
-              onEnterKeyDown={this._submitSearch}
-              ref={this._inputRef}
-            />
-          </Kb.Box2>
-          <Kb.Box2 direction="horizontal" gap="tiny" style={styles.resultsContainer}>
-            {this.props.inProgress && <Kb.ProgressIndicator style={styles.progress} />}
-            {this.props.totalResults > 0 && (
-              <Kb.Box2 direction="horizontal" gap="tiny">
-                <Kb.Text type="BodySmall" style={styles.results}>
-                  {this.props.selectedResult} of {this.props.totalResults}
-                </Kb.Text>
-                <Kb.Icon
-                  color={Styles.globalColors.black_50}
-                  fontSize={16}
-                  onClick={this.props.onUp}
-                  type="iconfont-arrow-up"
-                />
-                <Kb.Icon
-                  color={Styles.globalColors.black_50}
-                  fontSize={16}
-                  onClick={this.props.onDown}
-                  type="iconfont-arrow-down"
-                />
-              </Kb.Box2>
-            )}
+      <Kb.Box2 direction="vertical" fullWidth={true} style={this.props.style}>
+        <Kb.Box2 direction="horizontal" style={styles.outerContainer} fullWidth={true} gap="tiny">
+          <Kb.Box2 direction="horizontal" style={styles.inputContainer} fullWidth={true}>
+            <Kb.Box2 direction="horizontal" gap="xtiny" fullWidth={true} centerChildren={true}>
+              <Kb.Icon type="iconfont-search" color={Styles.globalColors.black_50} fontSize={16} />
+              <Kb.PlainInput
```

Comment:
when switching between convos you're searching in, the results stay consistent with the conversation but the query doesn't.